siad/main.go

```diff
@@ -168,6 +170,7 @@ func main() {
 	root.Flags().StringVarP(&globalConfig.Siad.Modules, "modules", "M", "cghrtw", "enabled modules, see 'siad modules' for more info")
 	root.Flags().BoolVarP(&globalConfig.Siad.AuthenticateAPI, "authenticate-api", "", false, "enable API password protection")
 	root.Flags().BoolVarP(&globalConfig.Siad.AllowAPIBind, "disable-api-security", "", false, "allow siad to listen on a non-localhost address (DANGEROUS)")
+	root.Flags().BoolVarP(&globalConfig.Siad.MemLogging, "enable-memlogging", "", false, "enable the logging of the sizes of various module data structures")
```

Comment:
It's not documented, therefore not a part of the compatibility promise.

Multiple people have been asking for runtime debugging options, and I'm inclined to agree with them actually. It's very annoying when you have something interesting happen to your node but it was running without the profile or debug flags, or you want a logging message that is DEBUG only. So I think it's actually better to be moving a lot of this stuff to the runtime, so you can disable and enable it in real time.